hr_skill/hr_skill_view.xml

```diff
@@ -11,15 +11,10 @@
         <field name="name">hr.employee.skill.form</field>
         <field name="model">hr.employee</field>
         <field name="inherit_id" ref="hr.view_employee_form"/>
-        <field name="type">form</field>
         <field name="arch" type="xml">
             <notebook position="inside">
                 <page string="Skills">
-                    <field name="skill_ids" nolabel="1">
```

Comment:
It would be nice to move the file in a sub directory 'views' to be conform to the Odoo guideline.